cypress/e2e/cloud/buckets.test.ts

```diff
@@ -113,6 +113,110 @@ describe('Explicit Buckets', () => {
+        {"name":"fsWrite","type":"field","dataType":"float"} ]`,
+      ],
+      schemaFile,
+      {type}
+    )
+
+    const event = {dataTransfer: {files: [testFile]}, force: true}
+    cy.getByTestID('dndContainer')
+      .trigger('dragover', event)
+      .trigger('drop', event)
+
+    cy.getByTestID('bucket-form-submit').click()
+
+    cy.getByTestID(`bucket-card explicit_bucket`)
+      .should('exist')
+      .within(() => {
+        cy.getByTestID('bucket-settings').click()
+      })
+    cy.getByTestID('accordion-header').click()
+
+    cy.getByTestID('measurement-schema-readOnly-panel-0')
+      .should('exist')
+      .within(() => {
+        cy.getByTestID('measurement-schema-name-0')
+          .contains('first schema file')
+          .should('exist')
+        cy.getByTestID('measurement-schema-download-button').click()
+        cy.readFile(`cypress/downloads/first_schema_file.json`)
+          .should('exist')
+          .then(fileContent => {
+            expect(fileContent[0].name).to.be.equal('time')
+            expect(fileContent[0].type).to.be.equal('timestamp')
+
+            expect(fileContent[1].name).to.be.equal('fsWrite')
+            expect(fileContent[1].type).to.be.equal('field')
+            expect(fileContent[1].dataType).to.be.equal('float')
+          })
+      })
+  })
+
+  it('should be able to create an explicit bucket and add schema file during editing', function() {
+    cy.getByTestID('Create Bucket').click()
+    cy.getByTestID('bucket-form-name').type('explicit_bucket')
+    cy.getByTestID('accordion-header').click()
+    cy.getByTestID('explicit-bucket-schema-choice-ID').click()
+
+    cy.getByTestID('bucket-form-submit').click()
+
+    cy.getByTestID(`bucket-card explicit_bucket`)
+      .should('exist')
+      .within(() => {
+        cy.getByTestID('bucket-settings').click()
+      })
+    cy.getByTestID('accordion-header').click()
+
+    cy.getByTestID('measurement-schema-add-file-button').click()
+    cy.getByTestID('input-field').type('first schema file')
+
+    const schemaFile = 'valid.json'
+    const type = 'application/json'
+    const testFile = new File(
+      [
+        `[{"name":"time","type":"timestamp"},
+        {"name":"fsWrite","type":"field","dataType":"float"} ]`,
+      ],
+      schemaFile,
+      {type}
+    )
+
+    const event = {dataTransfer: {files: [testFile]}, force: true}
+    cy.getByTestID('dndContainer')
+      .trigger('dragover', event)
+      .trigger('drop', event)
+
+    cy.getByTestID('bucket-form-submit').click()
+
+    cy.getByTestID(`bucket-card explicit_bucket`)
+      .should('exist')
+      .within(() => {
+        cy.getByTestID('bucket-settings').click()
+      })
+    cy.getByTestID('accordion-header').click()
+
+    cy.getByTestID('measurement-schema-readOnly-panel-0')
+      .should('exist')
+      .within(() => {
+        cy.getByTestID('measurement-schema-name-0')
+          .contains('first schema file')
+          .should('exist')
+      })
```

Comment:
looks great!  please add a section for downloading the file and checking the contents (the same code as in the first test you wrote, above)